src/modules/i18n/dropdown.js

```diff
@@ -20,6 +22,16 @@ function LanguageDropdown(props) {
 		});
 	}
 
+	const dispatch = useDispatch();
+
+	useEffect(() => {
```

Comment:
@petrKavulok I suppose, you can remove this useEffect and just the useEffect on lines 31-33 is enough and should stay, since it will be triggered anyway (but test that please)
https://github.com/TeskaLabs/asab-webui/blob/b0c4090160a7462bc8a34521644d900d9963fd64/src/modules/i18n/dropdown.js#L31-L33